src/templates/partials/footer.hbs

```diff
@@ -29,3 +33,7 @@
     </div>
   </section>
 </footer>
+</ul>
+</div>
+</section>
+</footer>
```

Comment:
Yeah there was some sort of error with the grunt, I ran it again and the extras disappeared